### PR TITLE
Update contacting-cloudflare-support.mdx

### PR DESCRIPTION
**Pull request by TracyCloudflare**
### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [x] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [x] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/docs/support/contacting-cloudflare-support.mdx [hunk 1]
```diff
@@ -21,23 +21,23 @@ Cloudflare Support *cannot* perform the following actions:
...
+| [Support case](#getting-help-with-an-issue)<br/>Use to troubleshoot specific issues or errors. Response times depend on your plan.                                                                                 | Yes        | Yes      | Yes         | No\*        |
 | **Chat**<br/>Use to troubleshoot specific issues or errors. Response times depend on your plan.                                                                                                                      | Yes        | Yes      | No          | No          |
-| **Emergency Phone** or **Emergency Phone Callback**<br/>Use phone support to reach out during emergencies such as site outages or DDoS attacks.                                                                      | Yes        | No       | No          | No          |
+| **Emergency Phone** <br/>Use phone support to reach out during emergencies such as site outages or DDoS attacks.                                                                                                     | Yes        | Yes      | No          | No          |
```
Comment (ngayerie) [inline src/content/docs/support/contacting-cloudflare-support.mdx]: Hi @TracyCloudflare 
Any reason for this change here?

**Review by kodster28 — APPROVED** [review]

**Review by ngayerie — CHANGES_REQUESTED** [review]
Need to clarify the phone availability

File: src/content/docs/support/contacting-cloudflare-support.mdx (hunk 1 above)
Comment (TracyCloudflare) [reply src/content/docs/support/contacting-cloudflare-support.mdx]: Hi @ngayerie 
Per Michelle M., we aren't offering call backs any longer. I also changed "ticket" to "case," as we are doing with the move to SFSC.

Comment (ngayerie) [reply src/content/docs/support/contacting-cloudflare-support.mdx]: Yes, these are fine :)
It's the change from No to Yes for the emergency phone for Business that I'm wondering about

Comment (TracyCloudflare) [reply src/content/docs/support/contacting-cloudflare-support.mdx]: @ngayerie Ah! You are right. Fixed.

**Review by ngayerie — APPROVED** [review]

**Pull request merged**